app/resources/brave_generated_resources_ko.xtb

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" ?>
 <!DOCTYPE translationbundle>
 <translationbundle lang="ko">
-<translation id="3333887201479867589">Brave 리워드</translation>
+<translation id="3333887201479867589">Brave Rewards</translation>
 <translation id="7942180974165298215">Brave 광고 차단</translation>
-<translation id="6406506848690869874">Brave 동기화</translation>
+<translation id="6406506848690869874">동기화</translation>
 <translation id="120759863357279975"><ph name="EXTENSION_NAME">$1</ph> (extension ID "<ph name="EXTENSION_ID">$2<ex>abacabadabacabaeabacabadabacabaf</ex></ph>")은(는) Brave에서 허용되지 않습니다.</translation>
```

Comment:
Recently, I added suggestions for korean translation bug(https://github.com/brave/brave-browser/issues/2868) in transifex.com.
I assume that isn't accepted and not updated yet.

Reply:
strings were added in, thx.